dpbench/benchmarks/rodinia/gaussian/gaussian_initialize.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+LAMBDA = -0.01
+
+
+def initialize(size, types_dict=None):
```

Comment:
We don't need a default value for `types_dict`.

Reply:
removed default type for `types_dict`.